pkg/upgrademode/config.go

```diff
@@ -0,0 +1,40 @@
+package upgrademode
+
+import (
+	"github.com/SAP/stewardci-core/pkg/k8s"
+	"github.com/pkg/errors"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/system"
+)
+
+const (
+	upgradeModeConfigMapName = "steward-upgrade-mode"
+
+	upgradeModeKeyName = "upgradeMode"
+)
```

Comment:
We should move these constants into the API package, as they _are_ part of the API (for operators).

Reply:
done